aws/resource_aws_servicecatalog_portfolio.go

```diff
@@ -52,9 +51,8 @@ func resourceAwsServiceCatalogPortfolio() *schema.Resource {
 				ValidateFunc: validation.StringLenBetween(0, 2000),
 			},
 			"provider_name": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				ValidateFunc: validation.StringLenBetween(1, 20),
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Do we want to enforce [documented](https://docs.aws.amazon.com/servicecatalog/latest/dg/API_CreatePortfolio.html#servicecatalog-CreatePortfolio-request-ProviderName) max length of `50`?

Reply:
Added.